processing/src/test/java/io/druid/query/aggregation/AggregationTestHelper.java

```diff
@@ -85,15 +95,35 @@
 public class AggregationTestHelper
 {
   private final ObjectMapper mapper;
-  private final GroupByQueryQueryToolChest toolChest;
-  private final GroupByQueryRunnerFactory factory;
+  private final QueryToolChest toolChest;
+  private final QueryRunnerFactory factory;
 
   private final TemporaryFolder tempFolder;
 
-  public AggregationTestHelper(List<? extends Module> jsonModulesToRegister, TemporaryFolder tempFoler)
+  private AggregationTestHelper(
+      ObjectMapper mapper,
+      QueryToolChest toolchest,
+      QueryRunnerFactory factory,
+      TemporaryFolder tempFolder,
+      List<? extends Module> jsonModulesToRegister
+  )
   {
-    this.tempFolder = tempFoler;
-    mapper = new DefaultObjectMapper();
+    this.mapper = mapper;
+    this.toolChest = toolchest;
+    this.factory = factory;
+    this.tempFolder = tempFolder;
+
+    for(Module mod : jsonModulesToRegister) {
+      mapper.registerModule(mod);
```

Comment:
that is a copy/paste error from my side, it shouldn't cause any problems. let me fix that.